src/BenchmarkDotNet/Templates/WasmCsProj.txt

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="$SDKNAME$" DefaultTargets="BuildApp">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <TargetFramework>$TFM$</TargetFramework>
+    <TargetArchitecture>wasm</TargetArchitecture>
+    <TargetOS>Browser</TargetOS>
+    <RuntimePackDir>$RUNTIMEPACKDIR$</RuntimePackDir>
+    <AppDir>$(MSBuildThisFileDirectory)\bin\net5.0\browser-wasm\publish</AppDir>
+    <AssemblyName>$PROGRAMNAME$</AssemblyName>
+    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
+    $COPIEDSETTINGS$
+  </PropertyGroup>
+
+
+  <UsingTask TaskName="WasmAppBuilder" 
```

Comment:
we discussed this in our wasm sync with @lewing today and think we should copy the WasmAppBuilder logic into BDN for now until we can push a proper version up into the .NET SDK. I'll look at that.